### PR TITLE
Adds robust granular access

### PR DESCRIPTION
**Pull request by IamVelcroboy**
## About the PR
- Adds individual access and airlocks for Boxer, Clown, Library, Mime, Musician, Reporter, Psychologist, and Zookeeper 
- Adds Boxer and Reporter the the Theatre group of accesses
- Adds a Sec/lawyer maintenance hatch
- Adds a salvage locked external airlock
- Updates Rep Drobe and Boxing Drobe with appropriate access
- Removes redundant service from Janitor(was merged upstream)

## Why / Balance
Roles with specific spaces mapped for them should have reasonable expectation of privacy/security in those spaces.

## Technical details
n/a

## Media
n/a

## Breaking changes
n/a

**Changelog**
n/a Primarily a mapping change

### CODE REVIEW

File: Resources/Prototypes/Access/medical.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 - type: accessLevel
   id: Chemistry
   name: id-card-access-level-chemistry
-  
+
```
Comment (Colin-Tel) [inline Resources/Prototypes/Access/medical.yml:12]: consider un-touching this whitespace

File: Content.Shared/Access/Components/IdCardConsoleComponent.cs [hunk 2]
```diff
@@ -64,9 +66,15 @@ public WriteToTargetIdMessage(string fullName, string jobTitle, List<string> acc
...
+        "Library",  // DeltaV - Add Library access 
         "Maintenance",
         "Medical",
+        "Mime", // DeltaV - Add Mime access
+        "Musician", // DeltaV - Add Musician access
         "Paramedic", // DeltaV - Add Paramedic access
+        "Psychologist", // DeltaV - Add Psychologist access
+        "Quartermaster",
+        "Reporter", // DeltaV - Add Psychologist access
         "Quartermaster",
```
Comment (Colin-Tel) [inline Content.Shared/Access/Components/IdCardConsoleComponent.cs]:
```suggestion
        "Reporter", // DeltaV - Add Psychologist access
        "Quartermaster",
```
Duplicate access

**Review by Colin-Tel — CHANGES_REQUESTED** [review]
I think this is something worth trying, only a couple things to change.

File: Content.Shared/Access/Components/IdCardConsoleComponent.cs (hunk 2 above)
Comment (Colin-Tel) [reply Content.Shared/Access/Components/IdCardConsoleComponent.cs]: also a little weird the reporter access has that comment

File: Resources/Prototypes/Access/medical.yml (hunk 1 above)
Comment (IamVelcroboy) [reply Resources/Prototypes/Access/medical.yml:12]: There's nothing there. I'm not even sure how to undo that.

**Comment by IamVelcroboy** [comment]
Fixed the dupe/type but the white-space I can't figure out how to undo. I even deleted the file and copied over a fresh one. It just doesn't register that I've changed anything. So, gives me nothing to commit.

**Review by Colin-Tel — APPROVED** [review]

**Pull request merged**